src/main/java/taskmaster/task/EventTask.java

```diff
@@ -0,0 +1,53 @@
+package taskmaster.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * This class encapsulates the information necessary for an Event task.
+ */
+
+public class EventTask extends Task {
+    /** Time and Date of Event. **/
+    private LocalDateTime eventDate;
+
+    /**
+     * Constructor for an EventTask.
+     *
+     * @param taskName Name/Description of Task.
+     *
+     * @param eventDate Time and Date of Event.
+     */
+
```

Comment:
I think that there shouldn't be a new line here (based on the java docs on module website)